customer-commons/src/main/java/no/unit/nva/customer/ObjectMapperConfig.java

```diff
@@ -5,7 +5,7 @@
 
 public final class ObjectMapperConfig {
 
-    public static final ObjectMapper objectMapper = JsonUtils.objectMapper;
+    public static final ObjectMapper objectMapper = JsonUtils.objectMapperWithEmpty;
```

Comment:
Necessary for ensuring that all serializations (including tests follow the pattern of the handler). Probably this class should be removed and we should rely on the default handler serializer. (But not in this PR)